src/side_chain.cpp

```diff
@@ -598,15 +598,18 @@ void SideChain::print_status()
 	uint32_t total_blocks_in_window = 0;
 	uint32_t total_uncles_in_window = 0;
 
-	uint32_t our_blocks_in_window = 0;
-	uint32_t our_uncles_in_window = 0;
+	// each dot corresponds to m_chainWindowSize / 30 shares, with current values, 2160 / 30 = 72
+	std::vector<uint32_t> our_blocks_in_window(30, 0);
+	std::vector<uint32_t> our_uncles_in_window(30, 0);
```

Comment:
If it's fixed size, it's better to use std::array.

Reply:
Will have them convert to `std::array<uint32_t, 30> our_blocks_in_window{};` for zero-initialization